src/interface_r/interface_r.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
Is this file needed?

Reply:
Yes it's better to have a Rproj so you can do a lot of stuff in RStudio more easily. Many R projects have this included in the repo.

Reply:
Ah, yes, got this confused with the other .Rproj files. Not all are needed, of course.